lib/manageiq/appliance_console/external_httpd_authentication/external_httpd_configuration.rb

```diff
@@ -170,8 +170,8 @@ def valid_environment?
         true
       end
 
-      def valid_parameters?(ipaserver)
-        host_reachable?(ipaserver, "IPA Server")
+      def valid_parameters?(_ipaserver)
+        true
```

Comment:
![](https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExZzkxczg2eHNvb2lqNnJqbjU5dXJ0MXM3NjV5MGs2YXVjbnh4ZXN4bCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/13d2jHlSlxklVe/giphy.gif)

Reply:
I actually deleted this method on the first pass, but put it back with a true anyway.  I like the placeholder / pattern it provided where we could hook in validations if needed.